frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/workload-profile.tsx

```diff
@@ -14,24 +15,32 @@ import { VMSettingsField } from '../../types';
 import { getPlaceholder } from '../../utils/renderable-field-utils';
 import { nullOnEmptyChange } from '../../utils/utils';
 import { iGetFieldValue } from '../../selectors/immutable/field';
+import { iGetName } from '../../selectors/immutable/selectors';
 
 export const WorkloadProfile: React.FC<WorkloadProps> = React.memo(
   ({
-    userTemplates,
+    iUserTemplate,
     commonTemplates,
     workloadProfileField,
-    userTemplate,
     operatingSystem,
     flavor,
     cnvBaseImages,
     onChange,
   }) => {
-    const vanillaTemplates = immutableListToShallowJS(
-      concatImmutableLists(iGetLoadedData(commonTemplates), iGetLoadedData(userTemplates)),
-    );
+    const isUserTemplateLoaded = iGetIsLoaded(iUserTemplate);
+    const userTemplateName: string = isUserTemplateLoaded
```

Comment:
same - no need to resolve the name here

Reply:
why not? we use it on later on `getWorkloadProfiles`